MiniKeePass/Lock Screen/LockScreenManager.m

```diff
@@ -15,6 +15,12 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+// See Technical Q&A QA1838
+
+// NOTE: This will NOT work with the old UIActionSheet depreciated class.
+//       Also, the UIActionSheetAutoDismiss class is broken and doesn't auto dismiss
+//       unless fixed.  If fixed then it works.
+
```

Comment:
You could remove the note.  We shouldn't be using the old UIActionSheet class anymore.